### PR TITLE
Update README with ABF example.

### PR DESCRIPTION
**Pull request by InnocentBug**
Maybe we should also have a hoomd example here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,45 +20,87 @@ dynamics engine
 
 PySAGES also depends on [JAX](https://github.com/google/jax/), follow their installation
 guide to set it up.
+It is important to install your flavor of jaxlib before you install PySAGES.
+Make sure that you install the CUDA compatible flavor for your system or the CPU version, depending on your local setup.
+PySAGES will not install jaxlib.
```
Comment (pabloferz) [inline README.md:25]:
```suggestion
PySAGES also depends on [JAX](https://github.com/google/jax/), follow their installation
guide to set it up. *NOTE:* make sure you manually install jaxlib before PySAGES.
Depending on your local setup, you will have to install the jaxlib CPU version or the CUDA compatible flavor.
```

**Comment by pabloferz** [comment]
In the long run we should remove the examples from the README and link to the documentation instead. For now we could point the user to the examples folder for more examples (including HOOMD).

**Comment by InnocentBug** [comment]
I am not opposed to that idea.
I just think, having a short and nice example on the README can attract some users.
(I always like that.)
Ideally, embedding the example files would be perfect, but it seems like Github does not allow that.

**Comment by pabloferz** [comment]
I'd like to avoid cramping the README with too much information. But we should write a couple of jupyter notebooks and add (Google Collab for example) badges for opening them from the README.

**Comment by InnocentBug** [comment]
Sure, sounds good.
So for now, I would either say, we remove the example from the README?

**Comment by pabloferz** [comment]
We can remove the example and have the following:
```diff
- Here is an example of how to hook pysages to an OpenMM simulation of alanine dipeptide
+ If you want to learn more, you can also head over to the `examples` subfolder,
+ which contains scripts for running different HOOMD-blue and OpenMM biased simulations.
```

**Comment by InnocentBug** [comment]
Sure, maybe we explicitly point to one OpenMM and one HOOMD example.

In #97 Marcus found the HOOMD-blue example for umbrella sampling, but missed the OpenMM ABF example and used the README instead.
So we can point people exactly where they want to start. (Not just method but also Backend.)

Or we sort the examples by backend first and then by method.

**Comment by pabloferz** [comment]
We should definitely sort them. I'd say Backend -> Method -> System is a good way to organize them.

**Comment by pabloferz** [comment]
Let's fix this by removing the example and point generically to `examples` and when we organize the examples we amend this to make it more specific.

**Comment by InnocentBug** [comment]
Sounds good to me.
Both comments

**Review by pabloferz — APPROVED** [review]

**Pull request merged**